src/patternfly/components/Alert/sr.hbs

```diff
@@ -0,0 +1,6 @@
+<span class="{{#if sr--modifier}}{{sr--modifier}}{{/if}}"
```

Comment:
the screenreader doesn't need a modifier so you can just have this as 
    ```<span class="pf-u-sr-only">
     {{> @partial-block}}
     </span>```

Reply:
done